conf/messages-ecc.cy

```diff
@@ -235,4 +235,14 @@ cds.subscription.organisation-disclose-personal-details-consent.yes=Iawn - Rwyf
 cds.subscription.organisation-disclose-personal-details-consent.no=Na - Dangoswch fy rhif EORI yn unig
 
 subscription-journey.how-confirm-identity.nino.heading=Nodwch eich rhif Yswiriant Gwladol
-subscription-journey.how-confirm-identity.utr.heading=Nodwch eich Cyfeirnod Unigryw y Trethdalwr
\ No newline at end of file
+subscription-journey.how-confirm-identity.utr.heading=Nodwch eich Cyfeirnod Unigryw y Trethdalwr
```

Comment:
You not marking these new entries with "TRANSLATE" any more?

Reply:
Unfortunately this will be going to prod after testing today.

Product team accepted risk with going with English version for Welsh tex on this page, but I think if someone will try to access this page in Welsh on prod is better to display English version without TRANSLATE

Reply:
True.